src/controllers/dynakube/activegate/internal/proxy/reconciler.go

```diff
@@ -41,10 +43,11 @@ func (r *Reconciler) Reconcile() error {
 	return r.ensureDeleted(context.TODO(), r.dynakube)
 }
 
-func NewReconciler(client client.Client, apiReader client.Reader, dynakube *dynatracev1beta1.DynaKube) *Reconciler {
+func NewReconciler(client client.Client, apiReader client.Reader, scheme *runtime.Scheme, dynakube *dynatracev1beta1.DynaKube) *Reconciler {
```

Comment:
does it make sense to pass `scheme` in? because its always the same right? (this happens in multiple cases)
and for tests you don't have to use the contractor